### ISSUE TITLE
[SCREENREADER]: ARIA attributes MUST conform to valid values

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#7769]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

ARIA attributes MUST have valid values. In this case, the accordion has `aria-hidden=""`. This attribute cannot be empty. This attribute likely doesn't belong on the accordion, and should instead use `aria-expanded="value"` with either true or false (see Potential fix below).

This issue appears on:
1. Supplemental Claims — [https://preview-prod.vfs.va.gov/preview?nodeId=3011](https://preview-prod.vfs.va.gov/preview?nodeId=3011)
1. Higher-Level Reviews — [https://preview-prod.vfs.va.gov/preview?nodeId=3003](https://preview-prod.vfs.va.gov/preview?nodeId=3003)

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to navigate the page with contextual change relationships conveyed through my assistive tech.

## Environment

- Operating System: Windows (or any)
- Browser: Firefox (or any)
- Screenreading device: NVDA (or any)
- Server destination: staging/CMS web demo

## Steps to Recreate

1. Enter https://preview-prod.vfs.va.gov/preview?nodeId=3011 in browser, within the VA environment
2. Use the axe browser extension (in the VA VM, I used Firefox to install the extension) to scan the page
3. Verify that there is a critical issue for ARIA attributes MUST conform to valid values

## Possible Fixes (optional)

There is an accordion with the following code:

`<div id="collapsible_panel_item-8622" class="usa-accordion-content" aria-hidden="">`

This component should use `aria-expanded="value"` instead, with `true` for if it is open and false for if it is closed, as indicated in the [VA design system](https://design.va.gov/components/accordions).

## WCAG or Vendor Guidance (optional)

* [ARIA attributes MUST conform to valid values - dequeuniversity](https://dequeuniversity.com/rules/axe/3.0/aria-valid-attr-value)
* [ARIA states and properties](https://a11yproject.com/posts/ARIA-states/)
* [Accessible Rich Internet Applications (WAI-ARIA) 1.1 - Supported States and Properties](https://www.w3.org/TR/wai-aria-1.1/#states_and_properties)

**Comment by jenstrickland** [comment]
This issue is with the axe scan happening in the build process, creating a false positive. The code in the DOM is fine. cc @MickinSahni 🎉
